guides/common/modules/proc_modifying-a-smart-class-parameter-by-using-an-external-file.adoc

```diff
@@ -0,0 +1,140 @@
+		"id": 3,
+		"merge_default": false,
+		"merge_overrides": false,
+		*"override": false,* # Set the override value to `true`.
+			"override_value_order": "fqdn\nhostgroup\nos\ndomain",
+		*"override_values": [],* # Remove this line.
+			"override_values_count": 0,
+		"parameter": "content",
+		"parameter_type": "string",
+		"puppetclass_id": 3,
+		"puppetclass_name": "motd",
+		"required": false,
+		*"updated_at": "2017-02-07 11:56:55 UTC",* # Remove this line.
+			"use_puppet_default": false,
+		"validator_rule": null,
+		"validator_type": ""
+}
+----
+. Use the parameter ID `3` to get the information specific to the `motd` parameter and redirect the output to a file, for example, *output_file.json*.
++
+Example request:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+$ curl --header "Accept:application/json" \
+--header "Content-Type:application/json" --request GET \
+--user _My_User_Name_:__My_Password__ \
+https://_{foreman-example-com}_/api/smart_class_parameters/3 \
+| python3 -m json.tool > output_file.json
+----
+. Copy the file created in the previous step to a new file for editing, for example, `changed_file.json`:
++
+----
+$ cp output_file.json changed_file.json
+----
+. Modify the required values in the file.
+In this example, change the content parameter of the `motd` module, which requires changing the `override` option from `false` to `true`:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+{
+	"avoid_duplicates": false,
+		*"created_at": "2017-02-06 12:37:48 UTC",* # Remove this line.
+			*"default_value": "",* # Add a new value here.
+			"description": "",
+		"hidden_value": "",
+		"hidden_value?": false,
+		"id": 3,
+		"merge_default": false,
+		"merge_overrides": false,
+		*"override": false,* # Set the override value to `true`.
+			"override_value_order": "fqdn\nhostgroup\nos\ndomain",
+		*"override_values": [],* # Remove this line.
+			"override_values_count": 0,
+		"parameter": "content",
+		"parameter_type": "string",
+		"puppetclass_id": 3,
+		"puppetclass_name": "motd",
+		"required": false,
+		*"updated_at": "2017-02-07 11:56:55 UTC",* # Remove this line.
+			"use_puppet_default": false,
+		"validator_rule": null,
+		"validator_type": ""
+}
+----
+. After editing the file, verify that it looks as follows and then save the changes:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+{
+	"avoid_duplicates": false,
+		*"default_value": "_No Unauthorized Access Allowed_",*
+			"description": "",
+		"hidden_value": "",
+		"hidden_value?": false,
+		"id": 3,
+		"merge_default": false,
+		"merge_overrides": false,
+		*"override": true,*
+			"override_value_order": "fqdn\nhostgroup\nos\ndomain",
+		"override_values_count": 0,
+		"parameter": "content",
+		"parameter_type": "string",
+		"puppetclass_id": 3,
+		"puppetclass_name": "motd",
+		"required": false,
+		"use_puppet_default": false,
+		"validator_rule": null,
+		"validator_type": ""
+}
+----
+. Apply the changes to {ProjectServer}:
```

Comment:
Please reword. As commented before/somewhere else, "apply the changes" remindes me too much about using `foreman-installer` to configure Foreman. Maybe "Submit the file to {Project}"? Or "Send ..."?